charts/openshift-metering/values.yaml

```diff
@@ -578,9 +587,6 @@ presto:
         logLevel: info
         taskMaxWorkerThreads: null
         taskMinDrivers: null
-        logLevel: info
-        taskMaxWorkerThreads: null
-        taskMinDrivers: null
```

Comment:
Why were these coordinator config options removed?

Reply:
they're dupes, linting found them, see right above?

Reply:
Oh wow, I guess those were introduced when we moved the hive configuration to it's section.